include/deal.II/physics/transformations.h

```diff
@@ -0,0 +1,1177 @@
+// ---------------------------------------------------------------------
+//
+// Copyright (C) 2016 by the deal.II authors
+//
+// This file is part of the deal.II library.
+//
+// The deal.II library is free software; you can use it, redistribute
+// it, and/or modify it under the terms of the GNU Lesser General
+// Public License as published by the Free Software Foundation; either
+// version 2.1 of the License, or (at your option) any later version.
+// The full text of the license can be found in the file LICENSE at
+// the top level of the deal.II distribution.
+//
+// ---------------------------------------------------------------------
+
+#ifndef dealii__transformations_h
+#define dealii__transformations_h
+
+#include <deal.II/base/tensor.h>
+#include <deal.II/base/symmetric_tensor.h>
+
+DEAL_II_NAMESPACE_OPEN
+
+
+namespace Physics
+{
+
+  /**
+  * A collection of operations to assist in the transformation of tensor
+  * quantities from the reference to spatial configuration, and vice versa.
+  * These types of transformation are typically used to reexpress quantities
+  * measured or computed in configuration in terms of a second configuration.
+  *
+  * For convenience we will predefine some commonly referenced tensors and
+  * operations.
+  * Considering the position vector $\mathbf{X}$ in the referential (material)
```

Comment:
You are repeating text that you already have in the other namespace. To avoid these two texts become out of sync, I would simply say here "We will use the same notation for coordinates $x,X$ and transformations $\varphi$ as discussed for namespace ABC."